vector/src/test/java/im/vector/util/AutoCompletionModeTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.util
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class AutoCompletionModeTest {
+    @Test
+    fun userMode_empty() {
```

Comment:
I like to have unit tests name like that : 
methodName_ParamIn_ExceptedResult

getWithText_ClassicText_ShouldReturnUserMode